scene/3d/physics/soft_body_3d.cpp

```diff
@@ -78,8 +81,48 @@ void SoftBodyRenderingServerHandler::close() {
 	write_buffer = nullptr;
 }
 
-void SoftBodyRenderingServerHandler::commit_changes() {
-	RS::get_singleton()->mesh_surface_update_vertex_region(mesh, surface, 0, buffer);
+void SoftBodyRenderingServerHandler::fti_pump() {
+	buffer_prev = buffer;
+	aabb_prev = RS::get_singleton()->mesh_get_custom_aabb(mesh);
```

Comment:
I wouldn't try this, it could cause a rendering server sync.
Either cache locally, or you could just ignore the AABBs as the culling is pretty approximate anyway.